codeship-steps.yml

```diff
@@ -10,36 +10,44 @@
           command: './packages/devops/scripts/ci/validateBranchName.sh'
         - name: Validate tests
           command: './packages/devops/scripts/ci/validateTests.sh'
-    - type: parallel
+    - type: serial
       name: Test
       service: testing
       steps:
-        - name: Test admin-panel-server
-          command: './packages/devops/scripts/ci/testBackend.sh admin-panel-server'
-        - name: Test lesmis-server
-          command: './packages/devops/scripts/ci/testBackend.sh lesmis-server'
-        - name: Test meditrak-server
-          command: './packages/devops/scripts/ci/testBackend.sh meditrak-server'
-        - name: Test web-config-server
-          command: './packages/devops/scripts/ci/testBackend.sh web-config-server'
-        - name: Test psss-server
-          command: './packages/devops/scripts/ci/testBackend.sh psss-server'
-        - name: Test report-server
-          command: './packages/devops/scripts/ci/testBackend.sh report-server'
-        - name: Test entity-server
-          command: './packages/devops/scripts/ci/testBackend.sh entity-server'
-        - name: Test database
-          command: './packages/devops/scripts/ci/testBackend.sh database'
-        - name: Test data-api
-          command: './packages/devops/scripts/ci/testBackend.sh data-api'
-        - name: Test auth
-          command: './packages/devops/scripts/ci/testBackend.sh auth'
-        - name: Test indicators
-          command: './packages/devops/scripts/ci/testBackend.sh indicators'
-        - name: Test internal-dependencies
-          command: './packages/devops/scripts/ci/testInternalDependencies.sh'
-        - name: Test web-frontend
-          command: './packages/devops/scripts/ci/testFrontend.sh web-frontend'
+        - type: parallel
```

Comment:
Split the tests into 2 batches. Arguably I should've left the existing tests as batch 1 and just started a new batch, it might make it a bit more obvious where to extend the tests... but this is prettier 😝

Reply:
Nice! Yeah this is prettier!